src/libraries/System.Private.CoreLib/src/System/Collections/Hashtable.cs

```diff
@@ -267,13 +271,16 @@ public Hashtable(int capacity, float loadFactor)
 
             double rawsize = capacity / _loadFactor;
             if (rawsize > int.MaxValue)
-                throw new ArgumentException(SR.Arg_HTCapacityOverflow, nameof(capacity));
+                ThrowHelper.ThrowArgumentException(ExceptionResource.Arg_HTCapacityOverflow, ExceptionArgument.capacity);
```

Comment:
Nit: I would prefer it if we kept these enhancements in a separate PR from the fastmod changes.